js/ad_manager_vast.js

```diff
@@ -589,24 +588,14 @@ OO.Ads.manager(function(_, $) {
       var maxMedia = _.max(streams, function(v) { return parseInt(v.bitrate, 10); });
       this.vastAdUnit.maxBitrateStream = maxMedia && maxMedia.url;
       this.vastAdUnit.durationInMilliseconds = OO.timeStringToSeconds(firstLinearAd.linear.Duration) * 1000;
-      if (OO.supportedVideoTypes.webm) {
-        this.vastAdUnit.streamUrl = this._extractStreamForType(streams, "webm");
-      }
-      if (this.vastAdUnit.streamUrl == null && OO.supportedVideoTypes.mp4) {
-        this.vastAdUnit.streamUrl = this._extractStreamForType(streams, "mp4");
-      }
-      if (this.vastAdUnit.streamUrl == null && OO.supportedVideoTypes.m3u8) {
-        OO.log("extract m3u8 stream here");
-      }
       _.extend(this.vastAdUnit.data, firstLinearAd);
       this.vastAdUnit.data.tracking = firstLinearAd.linear.tracking;
-
+      addToTimeline(this.vastAdUnit, adLoaded);
```

Comment:
actually I think we should as it might be a non-linear ad and this doesn't look for stream url of non-linear.